datasets/adult/metadata.yaml

```
@@ -1,42 +1,171 @@
 #Generated automatically by pmlb/write_metadata.py
 dataset: adult
-description: None yet. See our contributing guide to help us add one.
-source: None yet. See our contributing guide to help us add one.
-publication: None yet. See our contributing guide to help us add one.
+description: Predictipn of if person make over $50K a year, based on 14 other features.
+source: >
+  http://mlr.cs.umass.edu/ml/datasets/Adult
+  https://www.openml.org/d/1590
+publication: >
+  Ron Kohavi, "Scaling Up the Accuracy of Naive-Bayes Classifiers: a Decision-Tree Hybrid", 
+  Proceedings of the Second International Conference on Knowledge Discovery and Data Mining, 1996
 task: classification
 target:
   type: binary
-  description: None yet. See our contributing guide to help us add one.
-  code: None yet. See our contributing guide to help us add one.
+  description: If a given adult makes over $50K a year
+  code: >
+    ' <=50K' = 0, 
+    ' >50K' = 1
 features: # list of features in the dataset
   - name: age
     type: continuous
-    description: null # optional but recommended, what the feature measures/indicates, unit
-    code: null # optional, coding information, e.g., Control = 0, Case = 1
-    transform: ~ # optional, any transformation performed on the feature, e.g., log scaled
+    description: age of adults
   - name: workclass
-    type: discrete
+    type: categorical
+    description: working class (mostly between government or self employed)
+    code: >
+      '?' = 0, 
+      'Federal-gov' = 1, 
+      'Local-gov' = 2, 
+      'Never-worked' = 3, 
+      'Private' = 4, 
+      'Self-emp-inc' = 5, 
+      'Self-emp-not-inc' = 6, 
+      'State-gov' = 7, 
+      'Without-pay' = 8
   - name: fnlwgt
     type: continuous
+    description: >
+      population totals, weighted according to many parameters, including race, sex, and age (see openML page
+      under Variable Description for more details)
   - name: education
-    type: discrete
+    type: ordinal
+    description: education level of adult
+    code: >
+      ' 10th' = 0, 
```

Can we remove the space here and the lines below?

```suggestion
      '10th' = 0, 
```